onadata/apps/logger/migrations/0006_add_validation_status_json_field_in_instance_table.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import jsonfield.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('logger', '0005_instance_xml_hash'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='instance',
+            name='validation_status',
+            field=jsonfield.fields.JSONField(default=dict),
```

Comment:
This is very slow on staging—I think it's causing Postgres to rewrite every `logger_instance` row? I believe we can avoid that by making the field nullable (and setting the default to null?)

Reply:
Can a JSONfield be nullable? 
Yep, we can make it nullable and test if it doesn't break something when retrieving data.

Reply:
As you suggested, I've changed the migration. 
Before
```
Running migrations:
  Rendering model states... DONE
  Applying logger.0006_add_validation_status_json_field_in_instance_table... OK

real	69m47.510s
user	0m7.072s
sys	0m0.908s
```
After
```
Running migrations:
  Rendering model states... DONE
  Applying logger.0006_add_validation_status_json_field_in_instance_table... OK

real	0m2.907s
user	0m1.572s
sys	0m1.068s
```

Tests were run on table with 7 millions records (same computer both i5 3.1 Ghz, 16 GB ram)